components/developer/fmtlib/test/results-all.master

```diff
@@ -31,13 +31,20 @@
       Start 16: unicode-test
 16/20 Test #16: unicode-test .....................   Passed
       Start 17: xchar-test
-17/20 Test #17: xchar-test .......................   Passed
+17/20 Test #17: xchar-test .......................***Failed
       Start 18: enforce-checks-test
 18/20 Test #18: enforce-checks-test ..............   Passed
       Start 19: posix-mock-test
 19/20 Test #19: posix-mock-test ..................   Passed
       Start 20: os-test
 20/20 Test #20: os-test ..........................   Passed
 
-100% tests passed, 0 tests failed out of 20
+90% tests passed, 2 tests failed out of 20
 
+
+The following tests FAILED:
+	  3 - chrono-test (Failed
+	 17 - xchar-test (Failed
+Errors while running CTest
+Output from these tests are in: $(@D)/Testing/Temporary/LastTest.log
+Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
```

Comment:
You have two newly failing tests. Did you check why they are failing now?

Reply:
Yes,they seem to fail because they expect my time zone to be +01:00 but it's +02:00 because of summer time.

Reply:
Then can you try to add somehing like that to the Makefile and recheck:
COMPONENT_TEST_ENV += TZ=UTC

Reply:
That helped, all tests pass now.